src/components/app/ProfileFilterNavigator.js

```diff
@@ -28,7 +28,7 @@ const options: ExplicitConnectOptions<{||}, StateProps, DispatchProps> = {
       ? getFormattedTimeLength(
           profileSelection.selectionEnd - profileSelection.selectionStart
         )
-      : undefined;
+      : ''; // Used to be undefined, but it seems Redux won't merge that
```

Comment:
This is less than ideal. It adds an empty list item at all times rather than not having one when there is no selection